WORKFLOW.md

```diff
@@ -0,0 +1,133 @@
+- workflow policy implements Harmonized Xroad tests document [Harmonized X-Road test environment](https://github.com/ria-ee/blob/master/HARMONIZED_TEST_ENVIRONMENT.md)
+
+## 3 Development model
+
+3.1	X-Road uses [Github](https://github.com/) and [Git](https://git-scm.com/) based version management.
+
+## 4	Repositories
+
+4.1	The following code and documentation repositories are used:
+- Master Test Repository - short name: `X-Road-tests`; hosted by: GitHub; managed by: RIA; purpose: release of X-Road automated tests; access: Estonia Head Architect and Finland Scrum Master has write access; read access: ALL.
+	- https://github.com/ria-ee/X-Road-tests
+
+- Finland and Estonia develop in fork of RIA-EE/X-Road-tests
+	- Finland fork
+		- https://github.com/vrk-kpa/X-Road-tests
+	- Estonia fork
+		- https://github.com/asaquality/X-Road-tests
+
+4.2	Partners can establish their own, additional repositories, for backup, software distribution or other purposes.
+
+## 5	Branching pattern
+
+5.1	Branching pattern follows the [Gitflow model](https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow). Two perpetual branches – `master` , `develop` and own `testfeature` branches.
+
+- `master` branch is used to release X-Road tests
+- `develop` branch is used to test development
+- `testfeature` branches are used to work on test features
+
+Main repository:
+- `master` branch https://github.com/ria-ee/X-Road-tests
+	- Tagged releases of Xroad-tests
+- `develop` branch https://github.com/ria-ee/X-Road-tests
+	- Integration branch Finland and Estonia
+	- Latest integrated test development source code 
+	- Make pull request to master for tagging release or milestone
+
+Fork repository:
+- `master` branch https://github.com/vrk-kpa/X-Road-tests and https://github.com/asaquality/X-Road-tests
+	- Tagged releases of Xroad-tests
+- `develop` branch https://github.com/vrk-kpa/X-Road-tests and https://github.com/asaquality/X-Road-tests 
+	- Country branch for test developme
+	- Latest country test development source code 
+	- Make pull request to develop for ria-ee/X-Road-tests
+
+## 6 Tagging
+
+6.1 The versions merged to X-Road-tests/master branch are tagged by Finnish / Estonian responsible persons.
+
+git tag -a 6.16.0 -m "X-Road-tests for Xroad version 6.16.0"
+
+The versions merged to X-Road-tests/develop branch are not tagged.
+
+## 7 Changelog management and Pull requests
+
+Changelog description is visible in pull requests.Pull requests from Partners made against the `X-Road-tests/master` branch MUST follow these conventions
+
+- Pull request name format is:
+
+Finnish-08-2017-Description-of-pull-request
+Estonian-07-2017-Description-of-pull-request
+
+- Description field must contain at least the changelist. Any relevant additional information should also be provided here.
+
+## 8 Submitting and accepting tests work
+
+8.1 Upon completion of tests development, submit a pull request to `X-Road-tests/master`.
```

Comment:
Should be 'X-Road-tests/develop', master branch is used only for tagged versions.